native/explorer/src/dataframe.rs

```diff
@@ -456,7 +456,7 @@ pub fn df_drop_duplicates(
     maintain_order: bool,
     subset: Vec<String>,
 ) -> Result<ExDataFrame, ExplorerError> {
-    let df = &data.resource.0;
+    let df: &DataFrame = &data.resource.0;
```

Comment:
I left some of these hints because my language server was having a bad time without it. Not sure why this is not working for all functions :/